src/components/OrderBy.component.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import {
+  TiArrowSortedDown,
+  TiArrowSortedUp,
+  TiArrowUnsorted,
+} from "react-icons/ti";
+import { useSearchParams } from "react-router-dom";
+import styled from "styled-components";
+
+import { Order_By } from "../graphql/generated/graphql";
```

Comment:
is there a reason why you get a different naming convention for Order_By?

Reply:
That Order_By is autogenerated by the codegen